.buildkite/scripts/steps/test/jest_parallel.sh

```diff
@@ -22,9 +24,28 @@ else
 fi
 
 export TEST_TYPE
-echo "--- downloading jest test run order"
-download_artifact jest_run_order.json .
-configs=$(jq -r 'getpath([env.TEST_TYPE]) | .groups[env.JOB | tonumber].names | .[]' jest_run_order.json)
+
+# Added section for tracking and retrying failed configs
+FAILED_CONFIGS_KEY="${BUILDKITE_STEP_ID}${TEST_TYPE}${JOB}"
+
+if [[ ! "$configs" && "${BUILDKITE_RETRY_COUNT:-0}" == "1" ]]; then
```

Comment:
is it checking here strictly for `BUILDKITE_RETRY_COUNT==1` because we don't want this behaviour to be active across several retries? So only activate if we retry exactly once?

Reply:
yes! is the same logic we use for the FTR for now :)